resource_tracker_v2/models/transformer.py

```diff
@@ -29,7 +29,7 @@ class Meta:
                                              related_name="consume_from_resource_group")
     consume_from_attribute_definition = ForeignKey(AttributeDefinition, null=True, blank=True, on_delete=SET_NULL,
                                                    related_name="consume_from_attribute_definition")
-    factor = FloatField(null=True, blank=True)
+    factor = FloatField(default=1)
```

Comment:
I cannot retrieve why it was not set like this in first place. But there is a reason !

Reply:
All tests passed